public/Lab-Files/Lesson02/Challenges/images.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    <h1>The perfect trip</h1>
+<p>I would go to Greece.</p>
+<img src="./images/Greece.jpeg" alt="Picture of villas in Greece looking over the Mediterrenean ocean">
+<p>I would see scenes like the Parthenon.</p>
+<img src="./images/parthenon2.jpeg" alt="Picture of the historic ruins the Parthenon in Greece"> 
+<p>And I would do things like enjoy the beautiful beaches.</p>
+<img src="./images/Greece3.jpeg" alt="Picture of the Mediterrenean ocean and underwater cave">  
+    
+    
+    
+</body>
+</html>
```

Comment:
Your code has no errors. Great job!
The only suggestion I'd give you, is to make sure you properly indent and space your code for easier readability.